gruntfile.js

```diff
@@ -29,13 +29,13 @@ module.exports = function (grunt) {
                 karmaTests: 'tests/unit/**/*.js'
             }
         },
-        jshint: {
-            options: {
-                jshintrc: '.jshintrc'
-            },
-            all: [
-                '<%= config.files.lint %>'
-            ]
+        eslint: {
+            all: {
+                options: {
+                    config: 'eslint.json'
+                },
+                src: '<%= config.files.lint %>'
+            }
         },
         jscs: {
```

Comment:
Yes, it replaces both: More reading here:
https://medium.com/@markelog/jscs-end-of-the-line-bc9bf0b3fdb2#.x0o1xqkk9